core/graph/src/test/java/uk/gov/gchq/gaffer/graph/GraphTest.java

```diff
@@ -1203,7 +1203,6 @@ public void shouldThrowExceptionIfGraphIdIsInvalid() throws Exception {
     @Test
     public void shouldBuildGraphUsingGraphIdAndLookupSchema() throws Exception {
         // Given
-        HashMapGraphLibrary.clear();
```

Comment:
Why are you removing this, before it is used. look @ line: 1266 👀

Reply:
It is no longer a static method.  a graphLibrary would have to be instantiated then cleared each time, which currently it doesn't do.  It works without clearing it.

Reply:
👀 HashMapGraphLibrary lines: 30-32.

Reply:
@m55624  ?